### PR TITLE
FW-2751 add playback speed control to audio bar

### PR DESCRIPTION
**Pull request by sahil97**
#### Definition of Done
- [ ] Playback speed changes according to the rate displayed when the button is clicked.

### CODE REVIEW

File: frontend/app_v2/src/components/Audiobar/AudiobarPresentationMobile.js [hunk 1]
```diff
@@ -25,20 +35,30 @@ function AudiobarPresentationMobile({ data, curTime, duration, isPlaying, onPlay
...
             </button>
           </div>
-          <div className="flex justify-between">
-            <div>{useIcon('Placeholder', 'w-6 h-6 fill-current inline-flex')}</div>
-            <div className="text-white text-center">{`${curTime} / ${duration}`}</div>
-            <a href={data?.downloadLink} className="text-white float-right">
-              {useIcon('Download', 'w-6 h-6 fill-current inline-flex')}
-            </a>
+          <div className="flex">
+            <div className="w-2/6"></div>
```
Comment (Cara-Barter) [inline frontend/app_v2/src/components/Audiobar/AudiobarPresentationMobile.js:39]: because this div closes right away, I'm not sure it is doing anything...

**Review by Cara-Barter — APPROVED** [review]
Great job!! just one minor detail then you're good to go!

File: frontend/app_v2/src/components/Audiobar/AudiobarPresentationMobile.js (hunk 1 above)
Comment (sahil97) [reply frontend/app_v2/src/components/Audiobar/AudiobarPresentationMobile.js:39]: Oh Okay, so as this is just an empty container with a variable width to center the next div (playback time). 
I used it as a replacement for the empty icon which was here before to make it more responsive.

Comment (Cara-Barter) [reply frontend/app_v2/src/components/Audiobar/AudiobarPresentationMobile.js:39]: ok, sounds good!  Good to merge then

**Pull request merged**